### PR TITLE
[sql-gen] Support returning in insert and update queries

### PR DESCRIPTION
**Pull request by jaylees14**
Support `RETURNING` in Postgres `UPDATE` and `INSERT` queries

### CODE REVIEW

File: src/it/scala/temple/containers/PostgresSpec.scala [hunk 1]
```diff
@@ -37,24 +56,16 @@ trait PostgresSpec extends FlatSpec with DockerTestKit with DockerPostgresServic
...
-          case FloatVariable(value)      => prep.setFloat(i, value)
-          case DoubleVariable(value)     => prep.setDouble(i, value)
-          case DateVariable(value)       => prep.setDate(i, value)
-          case TimeVariable(value)       => prep.setTime(i, value)
-          case DateTimeTzVariable(value) => prep.setTimestamp(i, value)
-          case BlobVariable(value)       => prep.setBytes(i, value)
-          case UUIDVariable(value)       => prep.setObject(i, value)
-        }
-    }
+    setValues(prep, values)
```
Comment (c-harding) [inline src/it/scala/temple/containers/PostgresSpec.scala:59]: Very nice

**Review by c-harding — APPROVED** [review]
Very nice

File: src/main/scala/temple/generate/database/PostgresGenerator.scala [hunk 2]
```diff
@@ -10,6 +10,7 @@ import temple.generate.database.ast._
 import temple.generate.utils.CodeTerm._
 
 import scala.Option.when
+import scala.collection.immutable.{AbstractSeq, LinearSeq}
```
Comment (willjvsmith) [inline src/main/scala/temple/generate/database/PostgresGenerator.scala]: These used?

**Review by willjvsmith — APPROVED** [review]
LGTM 🚀

**Pull request merged**